### DISCUSSION TITLE
[Building] Shovel Rack

### DISCUSSION THREAD

**Discussion by TheDcol** [sunflower-land/sunflower-land#2353] [Ideas]
A building that increases stock quantity for sand shovels AND potentially other types of shovels in the future.

Building: SHOVEL RACK
Supply: 1 per Player, Helios
Utility: increases TI shovel stock +25
Cost: 20 crabs, 200 wood, 20 stone, 1 coral, 5 gold

This is some crappy art idea i stole from google, someone make it better and fitting for sfl pls

![image](https://user-images.githubusercontent.com/108110065/226249881-dff22067-8848-4623-8037-bbf22e2b6dcf.png)

We can adjust costs on what people think.
"BUT DCOL THIS IS ONLY FOR DAVY JONES PEOPLE WTF" Well, some people  do not restock everyday, some don't have Davy Jones nft and want to dig 25 times over the course of 2 days without restocking. That's my argument.
That's it, this is the idea.

Thanks

**Comment by VZozul** [comment]
It would be fair to at least increase the TI shovels and drills in ToolShed .. But I'm all for putting SHOVEL RACK. Moreover, they need to understand that after the end of Iron Idol, not everyone will continue to dig 50 times, and one of the reasons is because for this you need to do a restock, and why do I need to do a restock if I set myself WareHouse and Toolshed. And instead I choose to make 25 holes, but if I had a SHOVEL RACK, I would do 50. And by the way, in your version, the increase is only Shovels, drills also need to be added there)

**Reply by TheDcol** [reply]
Ye could also have +5 drills

**Reply by Thebyber** [reply]
I love this idea!

**Comment by adamhannigan** [comment]
Nice idea!

Our intention for Treasure Island is to be a burn feature, not a revenue generating feature. I'm all for enabling players to burn more resources and crafting shovels! 🔥

**Reply by Chucrutty** [reply]
wen?

**Reply by BAIAShady** [reply]
Hello Adam! It has been two months since this proposal was last updated, and I wanted to iterate my support for the Shovel Rack idea. I believe it would be a valuable addition to the game, aligning with the burn feature concept of Treasure Island. The ability to increase TI shovel stock through this building would not only enhance the gameplay experience but also encourage resource utilization. I would like to encourage more community members to join in the discussion and express their thoughts on this proposal. Your positive response earlier gave me hope that this idea could be implemented. Looking forward to any updates or further progress on this proposal!

**Comment by eliasSFL** [comment]
I'm all for this idea! I don't restock everyday and the fact that I can only get 25 shovels per restock means that I can't do treasure island everyday. Having an option to increase my Sand Shovel and Sand Drill stocks by 100% will allow me to dig everyday while restocking every other day (which is how often I restock anyways)

**Comment by PedroGrochibro** [comment]
as usually i love dcol ideas, so wen shovel rack ??

**Comment by Jakesmoked-141829** [comment]
up
